app/org/sagebionetworks/bridge/models/accounts/StudyParticipant.java

```diff
@@ -172,7 +178,8 @@ public boolean equals(Object obj) {
                 && Objects.equals(id, other.id) && Objects.equals(languages, other.languages)
                 && Objects.equals(lastName, other.lastName) && Objects.equals(notifyByEmail, other.notifyByEmail)
                 && Objects.equals(password, other.password) && Objects.equals(roles, other.roles)
-                && Objects.equals(sharingScope, other.sharingScope) && Objects.equals(status, other.status);
+                && Objects.equals(sharingScope, other.sharingScope) && Objects.equals(status, other.status)
+                && Objects.equals(timeZone, other.timeZone);
```

Comment:
so this won't consider a UTC time and a local-time-with-timezone to be equal even if they represent the same instant (ignoring relativistic effects)? if that's the intended behavior, then cool